julia-mode.el

```diff
@@ -906,10 +883,6 @@ following commands are defined:
 
 (add-hook 'inferior-julia-mode-hook 'inferior-julia--initialize)
 
-;;;###autoload
-(defalias 'run-julia #'inferior-julia
-  "Run an inferior instance of `julia' inside Emacs.")
-
```

Comment:
I guess that we can remove this, but technically it is a breaking change, so we should mention it in the changelog.

Reply:
I'm actually going to add this back in. I did a quick survey of other external language packages with a REPL, and the `run-LANGUAGE` function seems like a convention (Haskell had it, and I think ess had it).

Reply:
Thanks for looking into this.